src/resolve/resolved-dns-cache.c

```diff
@@ -728,6 +729,13 @@ int dns_cache_put(
                         return 0;
         }
 
+        if (cache_mode == DNS_CACHE_MODE_NO_NEGATIVE) {
+            char key_str[DNS_RESOURCE_KEY_STRING_MAX];
```

Comment:
indentation! please follow https://systemd.io/CODING_STYLE#formatting, i.e. stick to multiples of 8ch

Reply:
Done